enhancements/installer/vsphere-ipi-zonal.md

```diff
@@ -511,12 +734,107 @@ type VCenter struct {
 	// +kubebuilder:validation:MinItems=1
 	Datacenters []string `json:"datacenters"`
 }
+
+// Host defines host VMs to generate as part of the installation.
+type Host struct {
+	// FailureDomain refers to the name of a FailureDomain as described in https://github.com/openshift/enhancements/blob/master/enhancements/installer/vsphere-ipi-zonal.md
```

Comment:
Probably want to explain this in more depth in situ rather than referring out to a doc

Reply:
@JoelSpeed first time updating a enhancement. This is what is current in the infra spec, so are you expecting this to be updated? I suppose that should really be a bug right?

Reply:
Didn't realise this was the current state, we should probably look to update these where they are non-breaking, can you create a bug?